src/test/java/org/apereo/openequella/tools/toolbox/CheckFilesRestTests.java

```diff
@@ -0,0 +1,203 @@
+/*
+ * Licensed to The Apereo Foundation under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * The Apereo Foundation licenses this file to you under the Apache License,
+ * Version 2.0, (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apereo.openequella.tools.toolbox;
+
+import static org.junit.Assert.*;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+import java.io.File;
+import java.io.IOException;
+
+import org.junit.Test;
+
+public class CheckFilesRestTests {
+
+  @Test
+  public void testPlaceholder() {
+    // Remove once other tests are confirmed
+  }
+
+	// TODO re-implement if REST mode is fixed.
+//	@Test
+//	public void testRun1() throws IOException {
+//		try {
+//			assertTrue(PingEquellaDriver.setup("testData/props/e2e-run1-attachments.properties")); // Confirm no issues
+//			assertTrue(PingEquellaDriver.run());
+//			assertTrue(PingEquellaDriver.finalizeRun());
+//
+//			//ALL results
+//			File allResults = new File(ReportManager.getInstance().getStdOutFilename());
+//			String filename = Config.getInstance().getTestDataDir()+"/outputSamplesToCompare/ping_attachments_junitTestsAttachmentRun1_all_stats_GOLD.csv";
+//			File allResultsToCompare = new File(filename);
+//			assertTrue(allResults.exists());
+//			assertTrue(allResultsToCompare.exists());
+//			assertTrue(PingUtils.mediumStrictCompare(allResults, allResultsToCompare));
```

Comment:
Are these tests intended to be commented out?

Reply:
yes, that's correct.  The `REST` CheckFiles mode was not converted to oeq-toolbox yet.